optimal_bidding/environments/utils.py

```diff
@@ -36,3 +41,52 @@ def _download_transition_maps(self):
             transition_maps[hour] = pd.read_csv(filepath)
 
         return transition_maps
+
+def consolidate_csvs(folder_path, csv_path):
+    """
+    This will take a folder with MMS csvs and create a new csv with just the demand and energy data.
+    """
+#     with open(csv_path) as output:
+#         writer = csv.writer(output)
+#         # write header of output file
+#         writer.writerow(["Timestamp", "RRP", "Total_Demand"])
```

Comment:
FYI you can write the output and inputs in the comments like this:
"""Something general about the function 

Args:
   inputs: ...
Return: 
    outputs: ...
"""

Reply:
will add proper docstrings in next PR